File: JavaScript-NT/src/currencies.js
function Currencies() {
};

Currencies.prototype.currenciesList = [ 'NZD', 'USD', 'Euro' ];

Currencies.prototype.conversionTable = {
	NZD: 1,
	USD: 0.76,
	Euro: 0.67
};

Currencies.prototype.getConversionRateFromNZD = function(currency) {
	return this.conversionTable[currency];
};

File: C#/RefactorMe/Currencies.cs
using System.Collections.Generic;

namespace RefactorMe
{
	public enum Currency
	{
		NZD,
		USD,
		Euro
	}


	public static class Currencies
	{
		private static Dictionary<Currency, decimal> ConversionTable = new Dictionary<Currency, decimal>
		{
			{ Currency.NZD, 1.0m },
			{ Currency.USD, 0.76m },
			{ Currency.Euro, 0.67m }
		};


		public static decimal GetConversionRateFromNZD(Currency currency)
		{
			return ConversionTable[currency];
		}
	}
}
